src/cpu/x64/jit_generator.hpp

```diff
@@ -673,6 +673,16 @@ class jit_generator : public Xbyak::CodeGenerator, public c_compatible {
         }
     }
 
+    void uni_vhsubps(const Xbyak::Xmm &x, const Xbyak::Xmm &x2,
+             const Xbyak::Operand &op) {
```

Comment:
Strange indent

Reply:
Is it clang-format doing this ? Could you align third argument for better readability ?